Docker/PlexAniSync/README.md

```diff
@@ -19,19 +20,34 @@ docker run -d \
   ghcr.io/rickdb/plexanisync:latest
 ```
 
+### docker custom setting file
+
+```
+docker run -d \
+  --name=plexanisync \
+  --restart unless-stopped \
+-v /etc/localtime:/etc/localtime:ro \
+-v /host/path/folder:/config \
+-e SETTINGS_FILE="/config/settings.ini" \
+
+
+  ghcr.io/rickdb/plexanisync:latest
+```
+
 ### Environment Variables
 
-| ID                          | Default                  | Required | Note                                                                                                                                                                         |
-| --------------------------- | ------------------------ | :------: | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| PLEX_SECTION                | Anime                    | &#10003; | The library where your anime resides                                                                                                                                         |
```

Comment:
I would mark the required fields with an asterisk and then write below the table that they're not required if you provide a custom settings file.

Additionally, `runsync.sh` should only call `settingsupdater.py` if the SETTINGS_FILE environment variable was not set.